src/content/en/progressive-web-apps/_index.yaml

```diff
@@ -70,9 +70,22 @@ landing_page:
     items:
     - heading: Fast
       description: >
-        53% of users will abandon a site if it takes longer than 3 seconds to
-        load! And once loaded, they expect them to be fast, no janky scrolling
-        or slow to respond interfaces.
+        <p>
+          Speed is at the foundation of building a Progressive Web
+          App. Achieving a speedy load helps lowering bounce rate and
+          increases conversion.
```

Comment:
This is a bit wordy. Try:

Speed is the foundation of a Progressive Web App. A speedy load lowers bounce rate and increases conversion.